File: Ticktalk/app/src/main/java/com/example/ticktalk/SignupActivity.java
package com.example.ticktalk;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import com.google.android.gms.tasks.OnCompleteListener;
import com.google.android.gms.tasks.Task;
import com.google.firebase.auth.AuthResult;
import com.google.firebase.auth.FirebaseAuth;

public class SignupActivity extends AppCompatActivity {

    private EditText email;
    private EditText name;
    private EditText password;
    private Button btnSignup;

    private final static String TAG = "MyTag";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_signup);

        Log.d(TAG,"호출됨");

        email.findViewById(R.id.signupactivity_edittext_email);
        name.findViewById(R.id.signupactivity_edittext_name);
        password.findViewById(R.id.signupactivity_edittext_password);
        btnSignup.findViewById(R.id.signupactivity_Btn_signup);

        final String getEmail = email.getText().toString();
        final String getName = name.getText().toString();
        final String getPassword = password.getText().toString();

        btnSignup.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if(getEmail == null || getName == null || getPassword == null){
                    return ;
                }else {

                    FirebaseAuth.getInstance().createUserWithEmailAndPassword(getEmail, getPassword)
                            .addOnCompleteListener(SignupActivity.this, new OnCompleteListener<AuthResult>() {
                                @Override
                                public void onComplete(@NonNull Task<AuthResult> task) {

                                }
                            });
                }
            }
        });
    }
}

File: login_/app/src/main/java/com/example/login/SignUpActivity.kt
package com.example.login

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import com.google.firebase.auth.FirebaseAuth
import kotlinx.android.synthetic.main.activity_sign_up.*

class SignUpActivity : AppCompatActivity() {
    private var firebaseauth: FirebaseAuth? = null

    var email: String? = null
    var password: String? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_sign_up)

        firebaseauth = FirebaseAuth.getInstance()

        email = sign_edit_email.text.toString()
        password = sign_edit_password.text.toString()

        Toast.makeText(this, "무사 접속", Toast.LENGTH_SHORT).show()

        signUp_btn.setOnClickListener {
            create()
        }

    }

    private fun create(){
        firebaseauth!!.createUserWithEmailAndPassword(
            sign_edit_email.text.toString(),
            sign_edit_password.text.toString()
        ).addOnCompleteListener(this) {
            if(it.isSuccessful) {
                val user=firebaseauth?.currentUser
                Toast.makeText(this,"success",Toast.LENGTH_SHORT).show()
            } else {
                Toast.makeText(this,"failed",Toast.LENGTH_SHORT).show()
            }
        }
    }


    /*private fun createUserId() {
        firebaseauth!!.createUserWithEmailAndPassword(
            sign_edit_email.text.toString(),
            sign_edit_password.text.toString()
        ).addOnCompleteListener(this) {
            if (it.isSuccessful) {
                val user = firebaseauth!!.currentUser
                (user)
                Toast.makeText(this, "성공했습니다", Toast.LENGTH_SHORT).show()
            } else {
                Toast.makeText(this, "실패했습니다", Toast.LENGTH_SHORT).show()
            }
        }

    }*/
}